patches/minecraft/net/minecraft/client/gui/GuiIngame.java.patch

```diff
@@ -1,6 +1,14 @@
 --- ../src-base/minecraft/net/minecraft/client/gui/GuiIngame.java
 +++ ../src-work/minecraft/net/minecraft/client/gui/GuiIngame.java
-@@ -379,7 +379,8 @@
+@@ -55,6 +55,7 @@
+ import net.minecraft.util.text.ITextComponent;
+ import net.minecraft.util.text.TextFormatting;
+ import net.minecraft.world.border.WorldBorder;
++import net.minecraftforge.common.ForgeHooks;
```

Comment:
Do not use imports in patches, instead use the fully qualified name where it is used.

Reply:
Right, sorry. I'll fix.

Reply:
Fixed in latest commit.